modules/nf-core/dshbio/exportsegments/main.nf

```diff
@@ -2,10 +2,10 @@ process DSHBIO_EXPORTSEGMENTS {
     tag "${meta.id}"
     label 'process_medium'
 
-    conda "bioconda::dsh-bio=2.3"
+    conda "bioconda::dsh-bio=2.1"
```

Comment:
I don't understand the downgrade here, is that a merge issue?

Reply:
I just needed a PR to test our self-hosted runners. So, nothing to see here 🙂